fe/fe-core/src/main/java/org/apache/doris/datasource/hive/event/AlterTableEvent.java

```diff
@@ -124,4 +142,18 @@ protected void process() throws MetastoreNotificationException {
                     debugString("Failed to process event"), e);
         }
     }
+
+    @Override
+    protected boolean canBeBatched(MetastoreEvent that) {
+        if (!isSameTable(that)) {
+            return false;
+        }
+        if (isRename() || isView()) {
+            return true;
+        }
+        if (that instanceof AlterTableEvent) {
+            return !((AlterTableEvent) that).isRename() && !((AlterTableEvent) that).isView();
+        }
```

Comment:
Suggest abstracting partition related classes to determine if it is a partition type. Otherwise, you may wonder which other types of Table types are identified, and if it is a create database, why batch can be done (although we know that database related events will not go here)